src/test/kotlin/org/rust/ide/refactoring/RsInlineFunctionTest.kt

```diff
@@ -1071,6 +1071,72 @@ class RsInlineFunctionTest : RsTestBase() {
 
             """)
 
+    fun `test inline function from different module with two single use statements`() = doTest("""
+            use foo::bar::test;
```

Comment:
Let's keep 4 spaces style for the test code. Like
fun `test foo`() = doTest("""
    use ...;
""")